chaos_genius/views/data_source_view.py

```diff
@@ -629,10 +629,11 @@ def get_schema_tables():
                 )
 
             ds_name = getattr(ds_data, "connection_type")
-            schema = None if SCHEMAS_AVAILABLE[ds_name] == False else schema
-
+            schema = (
+                None if (ds_data.is_third_party or SCHEMAS_AVAILABLE[ds_name] == False) else schema
```

Comment:
Comparison with `False` can be better written this way:
```suggestion
                None if (ds_data.is_third_party or not SCHEMAS_AVAILABLE[ds_name]) else schema
```